ext/RMagick/rmmain.c

```diff
@@ -1734,14 +1734,19 @@ version_constants(void)
 {
     const char *mgk_version;
     VALUE str;
+    VALUE nr;
     char long_version[1000];
+    size_t lib_version;
 
-    mgk_version = GetMagickVersion(NULL);
+    mgk_version = GetMagickVersion(&lib_version);
 
     str = rb_str_new2(mgk_version);
     rb_obj_freeze(str);
     rb_define_const(Module_Magick, "Magick_version", str);
 
+    nr = rb_int_new(lib_version);
+    rb_define_const(Module_Magick, "Magick_lib_version", nr);
```

Comment:
Not sure why it was done this way in the test, but we probably want this to be all caps. Also, I might name it something like `IMAGEMAGICK_VERSION`.

Reply:
I used the same convention that was used for `Magick_version`. I can change it to `IMAGEMAGICK_VERSION` if you would prefer that.

Reply:
Yeah, I'd lean towards the all caps, one. @Watson1978, do you know why these aren't in all caps? I wonder if the old constants are used, or we can deprecate them.